examples/demo-app/src/app.js

```diff
@@ -104,10 +106,20 @@ class App extends Component {
     // if we pass an id as part of the url
     // we ry to fetch along map configurations
     const {
-      params: {id} = {},
+      params: {id, provider} = {},
       location: {query = {}} = {}
     } = this.props;
 
+    if (provider) {
+      const providerHandler = getCloudProvider(provider);
+
+      if (providerHandler) {
+        this.props.dispatch(providerHandler.loadMap(query));
```

Comment:
this is actually not ideal.

If you want to load a map/data using a provider, you should dispatch an action as it follows:
```
this.props.dispatch(loadMap(query, provider));
```

modify the loadMap action to support provider (string) as param.

Reply:
Agree, you are right